engine/src/flutter/lib/ui/semantics.dart

```diff
@@ -962,6 +1025,318 @@ class SemanticsFlag {
   String toString() => 'SemanticsFlag.$name';
 }
 
+/// Semantics flags.
+class SemanticsFlags {
+  /// All flags default to `false` unless specified.
```

Comment:
nit: the first paragraph of a constructor dartdoc should say what it does, such as "Creates a set of semantics flags that describe various states of a widget". The nuances of arguments usually goes into a separate paragraph.